src/utils/helpers/fetchUserInfo.ts

```diff
@@ -3,14 +3,19 @@ import config from '../../config'
 import { UserActions } from '../../contexts/UserState'
 import { getRequest } from './fetchMethods'
 import getServerUrl from './endpoints/endpointUrlBuilder'
+import { usePrefs } from '../../contexts/SystemPrefs'
 
 interface SetUserInfoProps {
   dispatch: Dispatch<UserActions>
 }
 
 const fetchUserInfo = ({ dispatch }: SetUserInfoProps, logout: Function) => {
+  const { preferences } = usePrefs()
+
+  const managementPrefName = preferences?.systemManagerPermissionName || 'systemManager'
```

Comment:
By default, the permission name to access "Management" templates is `systemManager`, but this can be overridden by preferences.json if they want it to be called something different. (Or, say, they just wanted to make one of the existing permissions the "management" one as well)

Reply:
Ok - I see that's determinate on the back-end so no risk for the "managerPermissionName" to be changed to some other users permissions on the Front-end